### PR TITLE
Updated code_checks.sh with PR06

### LINKED ISSUE #28724: Fix PR06 issues in docstrings
Fix the docstrings where there is an error in a parameter format, eg string instead of str.
Current errors:
```
pandas.Series.to_json: Parameter "indent" type should use "int" instead of "integer"
pandas.Timestamp.ceil: Parameter "freq" type should use "str" instead of "string"
pandas.Timestamp.day_name: Parameter "locale" type should use "str" instead of "string"
pandas.Timestamp.floor: Parameter "freq" type should use "str" instead of "string"
pandas.Timestamp.month_name: Parameter "locale" type should use "str" instead of "string"
pandas.Timestamp.round: Parameter "freq" type should use "str" instead of "string"
pandas.Timedelta: Parameter "value" type should use "int" instead of "integer"
pandas.Timedelta: Parameter "value" type should use "str" instead of "string"
pandas.Timedelta.max: Parameter "value" type should use "int" instead of "integer"
pandas.Timedelta.max: Parameter "value" type should use "str" instead of "string"
pandas.Timedelta.min: Parameter "value" type should use "int" instead of "integer"
pandas.Timedelta.min: Parameter "value" type should use "str" instead of "string"
pandas.Timedelta.ceil: Parameter "freq" type should use "str" instead of "string"
pandas.Timedelta.floor: Parameter "freq" type should use "str" instead of "string"
pandas.Timedelta.round: Parameter "freq" type should use "str" instead of "string"
pandas.Period.asfreq: Parameter "freq" type should use "str" instead of "string"
pandas.Period.to_timestamp: Parameter "freq" type should use "str" instead of "string"
pandas.arrays.IntervalArray.from_arrays: Parameter "copy" type should use "bool" instead of "boolean"
pandas.arrays.IntervalArray.from_tuples: Parameter "copy" type should use "bool" instead of "boolean"
pandas.arrays.IntervalArray.from_breaks: Parameter "copy" type should use "bool" instead of "boolean"
pandas.arrays.IntervalArray.to_tuples: Parameter "na_tuple" type should use "bool" instead of "boolean"
pandas.Categorical.from_codes: Parameter "codes" type should use "int" instead of "integer"
pandas.Categorical.from_codes: Parameter "dtype" type should use "str" instead of "string"
pandas.SparseArray: Parameter "kind" type should use "int" instead of "integer"
pandas.read_table: Parameter "cache_dates" type should use "bool" instead of "boolean"
pandas.read_csv: Parameter "cache_dates" type should use "bool" instead of "boolean"
pandas.read_msgpack: Parameter "iterator" type should use "bool" instead of "boolean"
pandas.ExcelWriter: Parameter "path" type should use "str" instead of "string"
pandas.ExcelWriter: Parameter "engine" type should use "str" instead of "string"
pandas.ExcelWriter: Parameter "date_format" type should use "str" instead of "string"
pandas.ExcelWriter: Parameter "datetime_format" type should use "str" instead of "string"
pandas.read_json: Parameter "orient" type should use "str" instead of "string"
pandas.read_html: Parameter "flavor" type should use "str" instead of "string"
pandas.HDFStore.append: Parameter "append      " type should use "bool" instead of "boolean"
pandas.HDFStore.append: Parameter "min_itemsize" type should use "str" instead of "string"
pandas.HDFStore.append: Parameter "nan_rep     " type should use "str" instead of "string"
pandas.HDFStore.append: Parameter "encoding    " type should use "str" instead of "string"
pandas.HDFStore.append: Parameter "dropna      " type should use "bool" instead of "boolean"
pandas.read_spss: Parameter "path" type should use "str" instead of "string"
pandas.read_sql_query: Parameter "sql" type should use "str" instead of "string"
pandas.read_sql_query: Parameter "con" type should use "str" instead of "string"
pandas.read_sql_query: Parameter "index_col" type should use "str" instead of "string"
pandas.read_sql_query: Parameter "coerce_float" type should use "bool" instead of "boolean"
pandas.read_sql: Parameter "sql" type should use "str" instead of "string"
pandas.read_sql: Parameter "con" type should use "str" instead of "string"
pandas.read_sql: Parameter "index_col" type should use "str" instead of "string"
pandas.read_sql: Parameter "coerce_float" type should use "bool" instead of "boolean"
pandas.read_gbq: Parameter "reauth" type should use "bool" instead of "boolean"
pandas.read_gbq: Parameter "auth_local_webserver" type should use "bool" instead of "boolean"
pandas.read_stata: Parameter "convert_dates" type should use "bool" instead of "boolean"
pandas.read_stata: Parameter "convert_categoricals" type should use "bool" instead of "boolean"
pandas.read_stata: Parameter "encoding" type should use "str" instead of "string"
pandas.read_stata: Parameter "index_col" type should use "str" instead of "string"
pandas.read_stata: Parameter "convert_missing" type should use "bool" instead of "boolean"
pandas.read_stata: Parameter "preserve_dtypes" type should use "bool" instead of "boolean"
pandas.read_stata: Parameter "order_categoricals" type should use "bool" instead of "boolean"
pandas.read_stata: Parameter "iterator" type should use "bool" instead of "boolean"
pandas.io.stata.StataReader.data: Parameter "convert_dates" type should use "bool" instead of "boolean"
pandas.io.stata.StataReader.data: Parameter "convert_categoricals" type should use "bool" instead of "boolean"
pandas.io.stata.StataReader.data: Parameter "index_col" type should use "str" instead of "string"
pandas.io.stata.StataReader.data: Parameter "convert_missing" type should use "bool" instead of "boolean"
pandas.io.stata.StataReader.data: Parameter "preserve_dtypes" type should use "bool" instead of "boolean"
pandas.io.stata.StataReader.data: Parameter "order_categoricals" type should use "bool" instead of "boolean"
pandas.core.resample.Resampler.pipe: Parameter "func" type should use "str" instead of "string"
pandas.testing.assert_frame_equal: Parameter "check_index_type" type should use "str" instead of "string"
pandas.testing.assert_frame_equal: Parameter "check_column_type" type should use "str" instead of "string"
pandas.testing.assert_series_equal: Parameter "check_index_type" type should use "str" instead of "string"
pandas.testing.assert_index_equal: Parameter "exact" type should use "str" instead of "string"
pandas.api.types.union_categoricals: Parameter "sort_categories" type should use "bool" instead of "boolean"
pandas.api.types.union_categoricals: Parameter "ignore_order" type should use "bool" instead of "boolean"
pandas.api.types.is_list_like: Parameter "allow_sets" type should use "bool" instead of "boolean"
pandas.qcut: Parameter "q" type should use "int" instead of "integer"
pandas.qcut: Parameter "labels" type should use "bool" instead of "boolean"
pandas.merge_asof: Parameter "left_index" type should use "bool" instead of "boolean"
pandas.merge_asof: Parameter "right_index" type should use "bool" instead of "boolean"
pandas.merge_asof: Parameter "tolerance" type should use "int" instead of "integer"
pandas.merge_asof: Parameter "allow_exact_matches" type should use "bool" instead of "boolean"
pandas.to_numeric: Parameter "downcast" type should use "int" instead of "integer"
pandas.period_range: Parameter "start" type should use "str" instead of "string"
pandas.period_range: Parameter "end" type should use "str" instead of "string"
pandas.period_range: Parameter "periods" type should use "int" instead of "integer"
pandas.period_range: Parameter "freq" type should use "str" instead of "string"
pandas.period_range: Parameter "name" type should use "str" instead of "string"
pandas.timedelta_range: Parameter "start" type should use "str" instead of "string"
pandas.timedelta_range: Parameter "end" type should use "str" instead of "string"
pandas.timedelta_range: Parameter "periods" type should use "int" instead of "integer"
pandas.timedelta_range: Parameter "freq" type should use "str" instead of "string"
pandas.timedelta_range: Parameter "name" type should use "str" instead of "string"
pandas.timedelta_range: Parameter "closed" type should use "str" instead of "string"
pandas.infer_freq: Parameter "warn" type should use "bool" instead of "boolean"
pandas.interval_range: Parameter "periods" type should use "int" instead of "integer"
pandas.interval_range: Parameter "freq" type should use "str" instead of "string"
pandas.interval_range: Parameter "name" type should use "str" instead of "string"
pandas.eval: Parameter "parser" type should use "str" instead of "string"
pandas.eval: Parameter "engine" type should use "str" instead of "string"
pandas.util.hash_array: Parameter "encoding" type should use "str" instead of "string"
pandas.util.hash_array: Parameter "hash_key" type should use "str" instead of "string"
pandas.util.hash_pandas_object: Parameter "index" type should use "bool" instead of "boolean"
pandas.util.hash_pandas_object: Parameter "encoding" type should use "str" instead of "string"
pandas.util.hash_pandas_object: Parameter "hash_key" type should use "str" instead of "string"
pandas.Index.shift: Parameter "freq" type should use "str" instead of "string"
pandas.CategoricalIndex: Parameter "dtype" type should use "str" instead of "string"
pandas.IntervalIndex.from_arrays: Parameter "copy" type should use "bool" instead of "boolean"
pandas.IntervalIndex.from_tuples: Parameter "copy" type should use "bool" instead of "boolean"
pandas.IntervalIndex.from_breaks: Parameter "copy" type should use "bool" instead of "boolean"
pandas.IntervalIndex.to_tuples: Parameter "na_tuple" type should use "bool" instead of "boolean"
pandas.MultiIndex.to_frame: Parameter "index" type should use "bool" instead of "boolean"
pandas.MultiIndex.to_frame: Parameter "name" type should use "str" instead of "string"
pandas.MultiIndex.sortlevel: Parameter "ascending" type should use "bool" instead of "boolean"
pandas.DatetimeIndex.indexer_at_time: Parameter "time" type should use "str" instead of "string"
pandas.PeriodIndex: Parameter "data" type should use "int" instead of "integer"
pandas.PeriodIndex: Parameter "freq" type should use "str" instead of "string"
pandas.PeriodIndex.to_timestamp: Parameter "freq" type should use "str" instead of "string"
pandas.core.groupby.GroupBy.pipe: Parameter "func" type should use "str" instead of "string"
pandas.core.groupby.DataFrameGroupBy.idxmax: Parameter "skipna" type should use "bool" instead of "boolean"
pandas.core.groupby.DataFrameGroupBy.idxmin: Parameter "skipna" type should use "bool" instead of "boolean"
pandas.core.groupby.DataFrameGroupBy.nunique: Parameter "dropna" type should use "bool" instead of "boolean"
pandas.core.groupby.DataFrameGroupBy.shift: Parameter "freq" type should use "str" instead of "string"
pandas.io.formats.style.Styler.set_table_attributes: Parameter "attributes" type should use "str" instead of "string"
pandas.api.extensions.ExtensionArray._from_sequence: Parameter "copy" type should use "bool" instead of "boolean"
pandas.api.extensions.ExtensionArray._from_sequence_of_strings: Parameter "copy" type should use "bool" instead of "boolean"
pandas.api.extensions.ExtensionArray.take: Parameter "indices" type should use "int" instead of "integer"
pandas.DataFrame.idxmax: Parameter "skipna" type should use "bool" instead of "boolean"
pandas.DataFrame.idxmin: Parameter "skipna" type should use "bool" instead of "boolean"
pandas.DataFrame.unstack: Parameter "fill_value" type should use "str" instead of "string"
pandas.DataFrame.to_json: Parameter "indent" type should use "int" instead of "integer"
```

### PR DESCRIPTION
**Pull request by Varun270**
- [ ✔] closes #28724
- [ ] tests added / passed
- [✔ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
you just need to add PR06, no need to remove any other codes / add blank lines

**Comment by Varun270** [comment]
> you just need to add PR06, no need to remove any other codes / add blank lines

Actually the PR was having merge conflicts so I resolved one, maybe during this process it might have add any kind of space.

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
please address https://github.com/pandas-dev/pandas/pull/43511#pullrequestreview-751895252

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Why did you remove GL02?

**Comment by Varun270** [comment]
> Why did you remove GL02?

Yeah, I really don't know how did that happen. I will fix this asap.

File: ci/code_checks.sh [hunk 1]
```diff
@@ -89,7 +89,7 @@ fi
 ### DOCSTRINGS ###
 if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
 
-    MSG='Validate docstrings (GL02, GL03, GL04, GL05, GL06, GL07, GL09, GL10, SS01, SS02, SS04, SS05, PR03, PR04, PR05, PR10, EX04, RT01, RT04, RT05, SA02, SA03)' ; echo $MSG
+    MSG='Validate docstrings (GL02, GL03, GL04, GL05, GL06, GL07, GL09, GL10, SS01, SS02, SS04, SS05, PR03, PR04, PR05, PR06, PR10, EX04, RT01, RT04, RT05, SA02, SA03)' ; echo $MSG
     $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=GL02,GL03,GL04,GL05,GL06,GL07,GL09,GL10,SS02,SS04,SS05,PR03,PR04,PR05,PR10,EX04,RT01,RT04,RT05,SA02,SA03
```
Comment (MarcoGorelli) [inline ci/code_checks.sh:93]: need to add PR06 to this line as well

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Looks like there's at least a couple of PR06 errors which still need fixing up

```
Error: /home/runner/work/pandas/pandas/pandas/core/tools/numeric.py:27:PR06:pandas.to_numeric:Parameter "downcast" type should use "int" instead of "integer"
Error: /home/runner/work/pandas/pandas/pandas/core/arrays/sparse/array.py:248:PR06:pandas.arrays.SparseArray:Parameter "kind" type should use "int" instead of "integer"
```

**Comment by Varun270** [comment]
> Looks like there's at least a couple of PR06 errors which still need fixing up
> 
> ```
> Error: /home/runner/work/pandas/pandas/pandas/core/tools/numeric.py:27:PR06:pandas.to_numeric:Parameter "downcast" type should use "int" instead of "integer"
> Error: /home/runner/work/pandas/pandas/pandas/core/arrays/sparse/array.py:248:PR06:pandas.arrays.SparseArray:Parameter "kind" type should use "int" instead of "integer"
> ```

kind
Here should I change integer to int in both the list and default as well?

**Comment by mzeitlin11** [comment]
> Here should I change integer to int in both the list and default as well?

For the sparse case, "integer" means something different than "int" (is referring to the type of index to construct, not a datatype), so I don't think it should change. Maybe this is a false positive?

**Comment by MarcoGorelli** [comment]
hmm, yeah, this might be a bug in numpydoc..I'll get back to this, might be worth checking if it also happens with the latest release

**Comment by Varun270** [comment]
downcast
Similar thing with downcast parameter, What should I do here @mzeitlin11 ?

**Comment by mzeitlin11** [comment]
> Similar thing with downcast parameter, What should I do here @mzeitlin11 ?
You could look into what @MarcoGorelli was saying above - whether this is an issue with `numpydoc` that has been fixed in a later release, or if there is some other way to work around it.

**Comment by Varun270** [comment]
> hmm, yeah, this might be a bug in numpydoc..I'll get back to this, might be worth checking if it also happens with the latest release

Till then Can I work on issue#21749

**Comment by MarcoGorelli** [comment]
yes, of course!

**Comment by Varun270** [comment]
> yes, of course!

Please do assign it to me as the bot is not responding to the "take" command.

**Comment by MarcoGorelli** [comment]
multiple people can work on that issue, there's no need to be assigned

**Comment by MarcoGorelli** [comment]
I reported this to NumpyDoc in https://github.com/numpy/numpydoc/issues/341 

Perhaps we can just work around it with something like
```
kind : str
    Can be 'integer' or 'block', default is 'integer'.
```
until/if it's fixed upstream

**Comment by Varun270** [comment]
> I reported this to NumpyDoc in [numpy/numpydoc#341](https://github.com/numpy/numpydoc/issues/341)
> 
> Perhaps we can just work around it with something like
> 
> ```
> kind : str
>     Can be 'integer' or 'block', default is 'integer'.
> ```
> 
> until/if it's fixed upstream



> I reported this to NumpyDoc in [numpy/numpydoc#341](https://github.com/numpy/numpydoc/issues/341)
> 
> Perhaps we can just work around it with something like
> 
> ```
> kind : str
>     Can be 'integer' or 'block', default is 'integer'.
> ```
> 
> until/if it's fixed upstream

So should I create a PR to add the changes that you suggested?

**Comment by MarcoGorelli** [comment]
you can update this one

**Comment by Varun270** [comment]
> you can update this one

alright will do it

**Comment by Varun270** [comment]
@MarcoGorelli I pushed the changes can you tell me what's wrong?

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
You'll also need to fix `downcast`

(and fix the merge conflicts)

File: pandas/core/tools/numeric.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def to_numeric(arg, errors="raise", downcast=None):
         - If 'raise', then invalid parsing will raise an exception.
         - If 'coerce', then invalid parsing will be set as NaN.
         - If 'ignore', then invalid parsing will return the input.
-    downcast : {'integer', 'signed', 'unsigned', 'float'}, default None
+    downcast : {'int', 'signed', 'unsigned', 'float'}, default None
```
Comment (MarcoGorelli) [inline pandas/core/tools/numeric.py:51]: needs to be a fix similar to what you did in pandas/core/arrays/sparse/array.py, this won't work

Comment (Varun270) [reply pandas/core/tools/numeric.py:51]: Okay you mean something like this -
```
downcast : str
           Can be 'integer' or 'signed' or 'unsigned' or 'float', default is 'None'.
```

Comment (MarcoGorelli) [reply pandas/core/tools/numeric.py:51]: I'd suggest
```
downcast : str, default None
    Can be 'integer', 'signed', 'unsigned', or 'float'.
```

File: ci/code_checks.sh [hunk 3]
```diff
@@ -93,8 +93,10 @@ fi
 ### DOCSTRINGS ###
 if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
 
-    MSG='Validate docstrings (GL01, GL02, GL03, GL04, GL05, GL06, GL07, GL09, GL10, SS01, SS02, SS03, SS04, SS05, PR03, PR04, PR05, PR08, PRO9, PR10, EX04, RT01, RT04, RT05, SA02, SA03)' ; echo $MSG
-    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=GL01,GL02,GL03,GL04,GL05,GL06,GL07,GL09,GL10,SS02,SS03,SS04,SS05,PR03,PR04,PR05,PR08,PR09,PR10,EX04,RT01,RT04,RT05,SA02,SA03
+
+    MSG='Validate docstrings (GL02, GL03, GL04, GL05, GL06, GL07, GL09, GL10, SS01, SS02, SS04, SS05, PR03, PR04, PR05, PR06, PR10, EX04, RT01, RT04, RT05, SA02, SA03)' ; echo $MSG
+    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=GL02,GL03,GL04,GL05,GL06,GL07,GL09,GL10,SS02,SS04,SS05,PR03,PR04,PR05,PR06,PR10,EX04,RT01,RT04,RT05,SA02,SA03
+
```
Comment (MarcoGorelli) [inline ci/code_checks.sh:99]: just to make the diff smaller
```suggestion
    MSG='Validate docstrings (GL02, GL03, GL04, GL05, GL06, GL07, GL09, GL10, SS01, SS02, SS04, SS05, PR03, PR04, PR05, PR06, PR10, EX04, RT01, RT04, RT05, SA02, SA03)' ; echo $MSG
    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=GL02,GL03,GL04,GL05,GL06,GL07,GL09,GL10,SS02,SS04,SS05,PR03,PR04,PR05,PR06,PR10,EX04,RT01,RT04,RT05,SA02,SA03
```

File: ci/code_checks.sh [hunk 4]
```diff
@@ -93,8 +93,8 @@ fi
 ### DOCSTRINGS ###
 if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
 
-    MSG='Validate docstrings (GL01, GL02, GL03, GL04, GL05, GL06, GL07, GL09, GL10, SS01, SS02, SS03, SS04, SS05, PR03, PR04, PR05, PR08, PRO9, PR10, EX04, RT01, RT04, RT05, SA02, SA03)' ; echo $MSG
-    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=GL01,GL02,GL03,GL04,GL05,GL06,GL07,GL09,GL10,SS02,SS03,SS04,SS05,PR03,PR04,PR05,PR08,PR09,PR10,EX04,RT01,RT04,RT05,SA02,SA03
+    MSG='Validate docstrings (GL02, GL03, GL04, GL05, GL06, GL07, GL09, GL10, SS01, SS02, SS04, SS05, PR03, PR04, PR05, PR06, PR10, EX04, RT01, RT04, RT05, SA02, SA03)' ; echo $MSG
+    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=GL02,GL03,GL04,GL05,GL06,GL07,GL09,GL10,SS02,SS04,SS05,PR03,PR04,PR05,PR06,PR10,EX04,RT01,RT04,RT05,SA02,SA03
```
Comment (MarcoGorelli) [inline ci/code_checks.sh:97]: I don't think you did the merge quite correctly - can you try again, and make sure that you only add PR06 (and not remove anything else, nor add any blank lines)?

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

File: ci/code_checks.sh (hunk 4 above)
Comment (Varun270) [reply ci/code_checks.sh:97]: Do you mean I have to add GL01, SS03, PR08, and PR09? Since PR06 is already added.

Comment (MarcoGorelli) [reply ci/code_checks.sh:97]: the diff shows that your PR removes GL01, SS03, PR08, and PR09 - these should not be removed, the only thing this PR should do to this file is to add PR06

Comment (Varun270) [reply ci/code_checks.sh]: Done!!

File: ci/code_checks.sh [hunk 5]
```diff
@@ -93,8 +93,9 @@ fi
 ### DOCSTRINGS ###
 if [[ -z "$CHECK" || "$CHECK" == "docstrings" ]]; then
 
-    MSG='Validate docstrings (GL01, GL02, GL03, GL04, GL05, GL06, GL07, GL09, GL10, SS01, SS02, SS03, SS04, SS05, PR03, PR04, PR05, PR08, PRO9, PR10, EX04, RT01, RT04, RT05, SA02, SA03)' ; echo $MSG
-    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=GL01,GL02,GL03,GL04,GL05,GL06,GL07,GL09,GL10,SS02,SS03,SS04,SS05,PR03,PR04,PR05,PR08,PR09,PR10,EX04,RT01,RT04,RT05,SA02,SA03
+    MSG='Validate docstrings (GL01, GL02, GL03, GL04, GL05, GL06, GL07, GL09, GL10, SS01, SS02, SS03, SS04, SS05, PR03, PR04, PR05, PR06, PR08, PR09, PR10, EX04, RT01, RT04, RT05, SA02, SA03)' ; echo $MSG
+    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=GL01,GL02,GL03,GL04,GL05,GL06,GL07,GL09,GL10,SS02,SS03,SS04,SS05,PR03,PR04,PR05,PR06,PR08,PR09,PR10,EX04,RT01,RT04,RT05,SA02,SA03
+
```
Comment (MarcoGorelli) [inline ci/code_checks.sh:98]:
```suggestion
    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=GL01,GL02,GL03,GL04,GL05,GL06,GL07,GL09,GL10,SS02,SS03,SS04,SS05,PR03,PR04,PR05,PR06,PR08,PR09,PR10,EX04,RT01,RT04,RT05,SA02,SA03
```

File: ci/code_checks.sh [hunk 6]
```diff
@@ -93,8 +93,9 @@ fi
...
     RET=$(($RET + $?)) ; echo $MSG "DONE"
```
Comment (MarcoGorelli) [inline ci/code_checks.sh:99]:
```suggestion
    $BASE_DIR/scripts/validate_docstrings.py --format=actions --errors=GL01,GL02,GL03,GL04,GL05,GL06,GL07,GL09,GL10,SS02,SS03,SS04,SS05,PR03,PR04,PR05,PR06,PR08,PR09,PR10,EX04,RT01,RT04,RT05,SA02,SA03
    RET=$(($RET + $?)) ; echo $MSG "DONE"
```

**Review by MarcoGorelli — APPROVED** [review]
looks good to me pending green

**Comment by jreback** [comment]
thanks @Varun270

**Comment by MarcoGorelli** [comment]
looks like this wasn't green
```
Error: /home/runner/work/pandas/pandas/pandas/core/frame.py:1104:PR06:pandas.DataFrame.to_string:Parameter "header" type should use "str" instead of "string"
```

**Pull request merged**